src/views/Home/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react'
+import { ScrollView, View } from 'react-native'
+import styled from 'styled-components/native'
+
+import UserPicture from '@assets/images/user_pictures/girl_1.png'
+
+import { spaces, ThemeAttributes } from '@src/styles/theme'
+
+import Credits from '@components/Credits'
+import Error from '@components/Error'
+import Layout from '@components/Layout'
+import Loader from '@components/Loader'
+import Text from '@components/Text'
+
+import SpotlightCard from './components/SpotlightCard'
+import ArticleCard from './components/ArticleCard'
+
+import WatchOption from './WatchOption'
+
+import type { News } from './types'
+
+const WatchOptionWrapper = styled.View`
+  flex-direction: row;
+  margin-top: ${spaces.SMALL}px;
+  margin-bottom: ${spaces.XX_LARGE}px;
+  margin-left: ${({ theme }: { theme: ThemeAttributes }) =>
+    theme.mode === 'light' ? `${spaces.XX_SMALL}px` : `${spaces.NONE}px`};
+`
+
+const Home = ({ navigation }: { navigation: any }) => {
+  const [news, setNews] = useState<News[]>([])
+  const [shouldDisplayError, setShouldDisplayError] = useState<boolean>(false)
+  const [isDataFetching, setIsDataFetching] = useState<boolean>(false)
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setIsDataFetching(true)
+        // eslint-disable-next-line no-undef
+        const response = await fetch(
+          'https://api.betaseries.com/news/last?key=72ede40de1b2&limit=15'
```

Comment:
Isn't there a way to fix this eslint error?

Reply:
Avoid putting the key in hard
Create a file that contains this kind of data (which you will obviously add in the .gitignore file)